pyblish_lite/window.py

```diff
@@ -627,6 +627,20 @@ def on_item_toggled(self, index, state=None):
 
         index.model().setData(index, state, model.IsChecked)
 
+        data = {"new_value": state, "old_value": not state}
+        if index.data(model.Type) == "instance":
+            instance = self.data["models"]["instances"].items[index.row()]
```

Comment:
There was a timeslip, a metaphysical anomaly and a black panther, and suddenly we're in the future where this still hasn't been looked at. I'll tune the time and space machine here at home and try to take another look this weekend.